exercises/verify.cpp

```diff
@@ -9,10 +9,24 @@ The number 3 is [not] present in the array.
 #include <iostream>
 using namespace std;
 
-int main()
-{
+int main() {
   // placeholder
-  int N[10] = [ 3, 4, 5, 1, 2, 3, 4, 9, 13, 0 ];
+  int N[10] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
+  int verify;
+  bool isPresent = false;
+
+  cout << "Insert number: ";
+  cin >> verify;
+  
+  for (int i=0; i<10 && !isPresent; i++) {
+    isPresent = N[i] == verify;
+  }
+
+  cout << "The number " << verify << " is";
+  if (!isPresent) {
+    cout << " not"; 
+  }
+  cout << " present in the array.\n";
```

Comment:
- `isPresent` non è necessario, puoi benissimo inserire dentro il ciclo for() un "return" per uscire dal ciclo, similmente a questa PR https://github.com/Helias/git-course-exercises/pull/5
o qui https://github.com/Helias/git-course-exercises/pull/52/files

- cerca di spezzettare il codice in più funzioni come in [questa PR](https://github.com/Helias/git-course-exercises/pull/52/files)